howto/diagnose-with-tracing/azure-monitor.md

```diff
@@ -1,28 +1,55 @@
-# Set up distributed tracing with Azure Monitor
+# Set up distributed tracing with Application insights
 
-Dapr integrates with Application Monitor through OpenTelemetry's default exporter along with a dedicated agent known as [Local Forwarder](https://docs.microsoft.com/en-us/azure/azure-monitor/app/opencensus-local-forwarder).
+Dapr integrates with Application Insights through OpenTelemetry's default exporter along with a dedicated agent known as [Local Forwarder](https://docs.microsoft.com/en-us/azure/azure-monitor/app/opencensus-local-forwarder).
 
-## How to configure distributed tracing with Azure Monitor
+## How to configure distributed tracing wit Application insights
 
-The following steps will show you how to configure Dapr to send distributed tracing data to Azure Monitor.
+The following steps will show you how to configure Dapr to send distributed tracing data to Application insights.
 
-### Setup Azure Monitor
+### Setup Application insights
 
 1. First, you'll need an Azure account. Please see instructions [here](https://azure.microsoft.com/free/) to apply for a **free** Azure account.
 2. Follow instructions [here](https://docs.microsoft.com/en-us/azure/azure-monitor/app/create-new-resource) to create a new Application Insights resource.
+3. Get Application insights Intrumentation key from your application insights page
+4. Go to `Configure -> API Access`
+5. Click `Create API Key`
+6. Select all checkboxes under `Choose what this API key will allow apps to do:`
+   - Read telemetry
+   - Write annotations
+   - Authenticate SDK control channel
+7. Generate Key and get API key
 
 ### Setup the Local Forwarder
 

```

Comment:
@youngbupark we should have a note here mentioning that the dapr local forwarder image is for testing purposes only, and we do not support it in production scenarios. We should point to the dockerfile and say teams can use this to create their own image

Reply:
I added some notes.I did not use word `testing` purpose. Instead, I used the word `preview`.  but feel free to edit the file directly if you want :)

Reply:
Hmmm not sure if we should use the term `preview` if we don't have eventual plans to turn it into a supported feature. Don't want companies to take dependencies thinking we'll continue to develop it. But you're right, `testing` might also not be great.

@msfussell what are your thoughts?

Reply:
I believe the term preview is used here for LocalForawrder which is from ApplciationInsights team and thus preview is used to refer to it.